inspire_json_merger/utils/filterout_utils.py

```diff
@@ -24,8 +24,13 @@
 
 from __future__ import absolute_import, division, print_function
 
+from functools import partial
 
-def filter_out(conflicts_list, fields):
+from inspire_utils.record import get_value
+from pyrsistent import freeze, thaw
```

Comment:
Uuu, what is this? 👀

Reply:
Fancy https://github.com/tobgu/pyrsistent#freeze

I would have just used `copy.deepcopy` (https://docs.python.org/2.7/library/copy.html#copy.deepcopy) and let the copy get wasted xd

Reply:
something to have immutable data structures but where copies are more efficient as they share the unmodified pieces. We don't want to modify the input records (as that would make testing tricky) and doing a full `deepcopy` is wasteful. `freeze` takes a standard Python container and returns its immutable version, `thaw` does the opposite. All of this is used throughout `json-merger`.

Reply:
Hmm... maybe it's better use copy, as it's stdlib (internal to python), that way we avoid the extra dep. Can you check if that's enough?

Reply:
there is no extra dep, as `json-merger` is using it all over the place.

Reply:
👍